### PR TITLE
[Merged by Bors] - Update block logs

### LINKED ISSUE #1797: more logs in block builder / listener
When a block is published, the "I've created a block" log contains block_id, layer_id, #tx, #atxs, view_size, votes_size and AssATX.

All of this information should also appear in the "got new block" log. At the moment this log only contains block_id and layer_id. In addition, it should also include the `sender_id`, which is the node_id of the miner that created the block.

If possible, both logs above should also include the proof_counter.

_After_ testing, this changes should be applied to the `closed_tn` branch.

### PR DESCRIPTION
**Pull request by noamnelke**
## Motivation
Closes #1797 

## Changes
Change the logs per @barakshani's request. Also cleanup the way we format and query those logs in cloud tests.

## Test Plan
Updated the way we query for "block created".

### CODE REVIEW

**Comment by noamnelke** [comment]
bors try

**Comment by noamnelke** [comment]
Ignore bors 🙄 the build succeeded:
https://travis-ci.org/spacemeshos/go-spacemesh/builds/648950327?utm_source=github_status&utm_medium=notification

File: tests/queries.py [hunk 1]
```diff
@@ -230,57 +227,47 @@ def parseAtx(log_messages):
...
-        # blocks - list of all blocks, layers - map of blocks per layer
-        if id in node2blocks:
-            node2blocks[id]["blocks"].append(m)
-            if layer in node2blocks[id]["layers"]:
-                node2blocks[id]["layers"][layer].append(m)
-            else:
-                node2blocks[id]["layers"][layer] = [m]
-        else:
-            node2blocks[id] = {"blocks": [m], "layers": {m[7]: [m]}}
+        node2blocks[log.node_id].add_block_to_layer(log.layer_id)
```
Comment (antonlerner) [inline tests/queries.py]: this will not store all kinds of data that has helped me debug runs such as block id etc...

Comment (noamnelke) [reply tests/queries.py]: fixed

File: cmd/node/app_test.go [hunk 2]
```diff
@@ -463,11 +463,17 @@ func (suite *AppTestSuite) validateLastATXActiveSetSize(app *SpacemeshApp) {
 }
 
 func TestAppTestSuite(t *testing.T) {
+	if testing.Short() {
```
Comment (antonlerner) [inline cmd/node/app_test.go:466]: dont skip this

File: cmd/node/app_test.go [hunk 3]
```diff
@@ -463,11 +463,17 @@ func (suite *AppTestSuite) validateLastATXActiveSetSize(app *SpacemeshApp) {
...
+		t.Skip()
+	}
 	//defer leaktest.Check(t)()
 	suite.Run(t, new(AppTestSuite))
 }
 
 func TestShutdown(t *testing.T) {
+	if testing.Short() {
```
Comment (antonlerner) [inline cmd/node/app_test.go:474]: dont skip this

File: cmd/node/app_test.go (hunk 2 above)
Comment (noamnelke) [reply cmd/node/app_test.go:466]: in short tests? why?

File: cmd/node/app_test.go (hunk 3 above)
Comment (noamnelke) [reply cmd/node/app_test.go:474]: same - this is only for short tests, which is not what happens when running `make test`, it just makes doing a quick run (3.5 minutes) of the short tests possible.

Comment (noamnelke) [reply cmd/node/app_test.go:474]: I have to admit that I didn't mean to commit this, but since I already have I think it's a good thing.

**Review by antonlerner — APPROVED** [review]

**Comment by noamnelke** [comment]
bors merge

File: miner/block_builder.go [hunk 4]
```diff
@@ -274,11 +274,19 @@ func (t *BlockBuilder) createBlock(id types.LayerID, atxID types.AtxId, eligibil
...
+
+	t.Log.Event().Info("block created",
+		bl.Id(),
+		bl.LayerIndex,
+		log.String("miner_id", bl.MinerId().String()),
+		log.Int("tx_count", len(bl.TxIds)),
+		log.Int("atx_count", len(bl.AtxIds)),
+		log.Int("view_edges", len(bl.ViewEdges)),
+		log.Int("vote_count", len(bl.BlockVotes)),
+		bl.ATXID,
```
Comment (barakshani) [inline miner/block_builder.go:287]: is this short id?

File: sync/block_listener.go [hunk 5]
```diff
@@ -85,9 +85,20 @@ func (bl *BlockListener) handleBlock(data service.GossipMessage) {
...
+	bl.Log.With().Info("block received",
+		blk.Id(),
+		blk.LayerIndex,
+		blk.LayerIndex.GetEpoch(bl.LayersPerEpoch),
+		log.String("miner_id", blk.MinerId().String()),
+		log.Int("tx_count", len(blk.TxIds)),
+		log.Int("atx_count", len(blk.AtxIds)),
+		log.Int("view_edges", len(blk.ViewEdges)),
+		log.Int("vote_count", len(blk.BlockVotes)),
+		blk.ATXID,
```
Comment (barakshani) [inline sync/block_listener.go:99]: short id?

**Pull request closed without merging**